### ISSUE TITLE
[6.6.x] Sub categories of type folder are not visible with A11Y changes

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#7592]
Within SW6.6.x and feature flag {{ACCESSIBILITY_TWEAKS=1}} enabled sub categories of type folder / structuring element are not visible with A11Y changes. Probably we have that issue also in 6.7. but the verification is not possible because of that ticket [here|https://shopware.atlassian.net/browse/NEXT-40154].

*Steps to reproduce:*

# Checkout {{6.6.x}} branch and activate feature flag: {{ACCESSIBILITY_TWEAKS=1}}
# Create a Sub category {{HOME}} with the type: Structuring Element / Entry Point → activate it
# Create a new sub category for the category in step 2 with type: “Page / List” → activate it
# Move to Storefront and {{HOVER}} and /or Click on that category

*Expected behaviour:*

Expectations for SW 6.6.x and 6.7.x

* Hover of that category from step 2 should be possible (see image)
* Click on sub category from step 3 should be possible (via hover)
* Breadcrumb should be visible after clicking on sub category of step 3
* following acceptance test should be green: {{tests/acceptance/tests/Categories/NavigateThroughStorefrontMenu.spec.ts}}

*Actual behaviour:*

Within 6.6.x the sub menus can be created BUT they are not working in Storefront.

||*Correct*|| || ||
|!image-20250212-102022.png|width=559,height=347,alt="image-20250212-102022.png"!|!image-20250212-102037.png|width=980,height=434,alt="image-20250212-102037.png"!| |

||*False*|| || ||
|!image-20250212-102110.png|width=980,height=434,alt="image-20250212-102110.png"!| | |

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40641**



[created from NEXT-40641, comment 545264]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.3**

 This issue will be / was solved in Shopware version ***6.6.10.3***.



[created from NEXT-40641, comment 545661]

**Comment by timo-haas-gh** [comment]
The focus via tab should be possible aswell
